### ISSUE TITLE
"TypeError: body is null"  when mouseover on particle on layout.

### ISSUE THREAD

**Issue by marktaylor46** [gantry/gantry5#1154]
I checked this in a number of templates and G5.2.10 and G5CI.

If you mouseover the drag thumb on the layout there are JS errors appearing in the browser console every time you mouseover:
TypeError: body is null in xxxxxx/administrator/components/com_gantry5/js/main.js line 28146

![2016-02-18_15-39-31](https://cloud.githubusercontent.com/assets/11683218/13148737/77eec984-d656-11e5-9f93-1174018e4b18.png)
![2016-02-18_15-38-52](https://cloud.githubusercontent.com/assets/11683218/13148740/7ab55f16-d656-11e5-8fbd-381600a15919.png)
![2016-02-18_15-37-24](https://cloud.githubusercontent.com/assets/11683218/13148742/7d7fef18-d656-11e5-95ef-d884ef3678e2.png)

**Comment by w00fz** [comment]
Can't reproduce

**Comment by marktaylor46** [comment]
So, a little more information...

The errors above I found in firefox 44.0.2 in three different templates with both Gantry 5.2.10 and Gantry 5CI. I am using Windows 10 64bit.

I tried the same thing in Chrome and there are no errors in the console.

I tried the same thing in IE Edge and got a different error message:
"unable to get property 'appendChild" of undefined or null reference"

![2016-02-19_08-37-01](https://cloud.githubusercontent.com/assets/11683218/13170508/60876d26-d6e4-11e5-9267-8a298c381da1.png)
